react-native-reanimated.d.ts

```diff
@@ -380,6 +424,68 @@ declare module 'react-native-reanimated' {
       initialValue: T
     ): AnimatedValue<T>;
 
+    // reanimated2 functions
+    export function runOnUI<A extends any[], R>(fn: (...args: A) => R): (...args: Parameters<typeof fn>) => void;
+    export function processColor(color: number | string): number;
+
+    // reanimated2 hooks
+    // lets not use extends on the function generic, which causes it to be more specific
+    // but let's do it on the return type instead
+    export function useSharedValue<T>(
+        initialValue: T
+    ): T extends SharedValueType ? SharedValue<T> : never {
+      return {
+        value: initialValue
+      } as any;
```

Comment:
I’m not sure about this `any`

Reply:
I think you can remove function implementation from type definition :)

Reply:
Oh, yeah, silly me